api/v1/verticadb_webhook.go

```diff
@@ -1081,6 +1081,13 @@ func (v *VerticaDB) validateSandboxes(allErrs field.ErrorList) field.ErrorList {
 	mainClusterImage := v.Spec.Image
 	path := field.NewPath("spec").Child("sandboxes")
 	for i, sandbox := range sandboxes {
+		// check if we have empty sandbox names
+		if sandbox.Name == "" {
```

Comment:
I know it does not hurt to check this, but I don't think an empty value for the sandbox will be allowed without this check as it is set as a required parameter, but I am not 100% sure. Have you tried to set an empty value?

Reply:
I didn't try it. It shouldn't be allowed since the main cluster has the empty string as the sandbox name. If we sandbox one subcluster to an empty string, we would get an error from vcluster.